enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -2322,6 +2503,9 @@ func (p *PebbleCache) Start() error {
 	p.eg.Go(func() error {
 		return p.backgroundRepair(p.quitChan)
 	})
+	p.eg.Go(func() error {
+		return p.migrateData(p.quitChan)
```

Comment:
let's add a check here for err != nil + alert.UnexpectedEvent so we know if a migration did not finish successfully